### DISCUSSION TITLE
InputNumber: Support `roundingMode` for the InputNumber

### DISCUSSION THREAD

**Discussion by trandaison** [primefaces/community#611] [PrimeReact] [answered]
Since `InputNumber` is using `Intl.NumberFormat` for formating the value.
There are some `options` are not supported, such as [`roundingMode`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#roundingmode).

How about adding the `roundingMode` props for `InputNumber`?

**Comment by melloware** [comment, marked as answer]
GitHub Issue Created: https://github.com/primefaces/primereact/issues/5528
